### PR TITLE
Fix android multiline material input

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

**Review by savelichalex — COMMENTED** [review]
Are you sure about not using `useAutogrowTextView` for mobile platforms? Have you checked that on all platforms? Have you checked that in chat and browser?

**Comment by sertony** [comment]
> Are you sure about not using `useAutogrowTextView` for mobile platforms? Have you checked that on all platforms? Have you checked that in chat and browser?

yep, that also doesn't feel right to me...

**Comment by SAnatoliiS** [comment]
I have checked on all platforms (android/ios/web) in inputs, chat and browser.

**Comment by SAnatoliiS** [comment]
TextInput needs nothing to change it's height in multiline mode. But using `useAutogrow` leads to reset input value when word doesn't fits in line.

**Comment by savelichalex** [comment]
What about constrainedHeight? Does it still work?

File: kit/inputs/src/UIMaterialTextView/useAutogrow.ts [hunk 1]
```diff
@@ -1,88 +1,21 @@
...
 export function useAutogrow(
-    ref: React.Ref<TextInput>,
+    _ref: React.Ref<TextInput>,
     onContentSizeChangeProp: UIMaterialTextViewProps['onContentSizeChange'],
     onChangeProp: UIMaterialTextViewProps['onChange'],
-    multiline: UIMaterialTextViewProps['multiline'],
+    _multiline: UIMaterialTextViewProps['multiline'],
     numberOfLinesProp: UIMaterialTextViewProps['numberOfLines'],
-    onHeightChange?: OnHeightChange,
+    _onHeightChange?: OnHeightChange,
```
Comment (sertony) [inline kit/inputs/src/UIMaterialTextView/useAutogrow.ts:12]: I see this is used in so many places, hence it needs to be fixed differently...

**Pull request closed without merging**